### ISSUE TITLE
Can't find relative path

### ISSUE THREAD

**Issue by xaolanx** [vasujain275/rudra#2]
Does nix flake support relative path? I'm try rebuild my system using this repo but always sgow fetch error relative path:./module/nixCats

**Comment by vasujain275** [comment]
Did you change all the references of "vasu" to your username and all the references of "rudra" to your hostname?

**Comment by will-x86** [comment]
Change this line 
https://github.com/vasujain275/rudra/blob/c2d9dc0fbf63fb968995de3fdd385b94b7ae6eef/flake.nix#L12

to 
```
url = "path:/home/YOUR_USERNAME/rudra/modules/nixCats"; 
```

**Comment by McEazy2700** [comment]
Hi @vasujain275 running `sudo nix flake update` fixes it

**Comment by vasujain275** [comment]
@McEazy2700 Yes, currently I am working on updating the whole nixCats with new Kickstart Template and new nixCatsUtils version
